# This is a configuration file to set JPS planning non-optional parament
planning:
  planning_tree_resolution: 0.01 # [m]
  planning_timeout: 60000          # [s]
  JPS_timeout:      600000        # [ms]
  planner_verbose: true          # [print info or not]
  debug_verbose: false           # [print info or not]
  xDim: 1.8                      # [m]
  yDim: 1.8                      # [m]
  zDim: 1.8                      # [m]
  euclidean_distance_cutoff: 1.8 # [m]
  eps:  1.0                      # [heuristic gain]
  max_iteration: 10              # [maximum iteration times for getting optimal path]
  JPS_max_iteration: 10000        # [maximum iteration times for getting optimal path]
  search_radius: 3               # [increment of key value]
  potential_radius: 5            # [increment of key value]
  potential_weight: 0.5          # [potential field gain]
  heu_type: Manhattan          # heuristic type
  # heu_type: Euclidean          # heuristic type (Failed)
  # heu_type: Diagonal           # heuristic type

visualization:
  show_unoccupied: false         # show unoccupied voxel of the octree
  tree_points_scale: 0.02        # be same with the planning_tree_resolution
  expansions_points_scale: 0.01  # expansion nodes marker scale
  path_points_scale: 0.005       # path points marker scale
  point_scale: 0.015             # single point marker scale
  voxel_scale: 0.01              # single voxel marker scale